webapp/portlet/src/main/webapp/vue-apps/activity-reactions/components/ActivityReactions.vue

```diff
@@ -85,6 +87,10 @@ export default {
       // eslint-disable-next-line no-magic-numbers
       default: () => 30,
     },
+    disableButtons: {
```

Comment:
use simple word to define the whole component as disabled
```suggestion
    disabled: {
```